Formula/zellij.rb

```diff
@@ -0,0 +1,22 @@
+class Zellij < Formula
+  desc "Pluggable terminal workspace, with terminal multiplexer as the base feature"
+  homepage "https://zellij.dev"
+  url "https://github.com/zellij-org/zellij/archive/refs/tags/v0.8.0.tar.gz"
+  sha256 "850d1a68219debdb4e05f7f58e70b31f440425247caaa1b29b9be1b6c7869207"
+  license "MIT"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    assert_match "zellij #{version}", shell_output("#{bin}/zellij --version")
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
Actually I don't think this is possible. I was inspired by the tmux formula which currently only verifies the the program exits normally 

https://github.com/Homebrew/homebrew-core/blob/6464b8807b31a5dc781f9a9fe02e7e18f2729e43/Formula/tmux.rb#L71-L73

Reply:
You can dump the default configuration to stdout:
```
zellij setup --dump-config
```

Reply:
```sh
zellij generate-completion fish
```
This should probably be done in the install block. But default config seems like a reasonable option if nothing else is possible.

Reply:
Done